core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitManager.java

```diff
@@ -75,7 +75,7 @@ public ConsensusCommitManager(
     mutationOperationChecker = new ConsensusCommitMutationOperationChecker(tableMetadataManager);
   }
 
-  ConsensusCommitManager(DatabaseConfig databaseConfig) {
+  public ConsensusCommitManager(DatabaseConfig databaseConfig) {
```

Comment:
To make it easy to inherit `ConsensusCommitManager`. This change is helpful to utilize the new extension point.

Reply:
So, `protected` is more appropriate?
```suggestion
  protected ConsensusCommitManager(DatabaseConfig databaseConfig) {
```

Reply:
Thanks! I confirmed it works. Addressed it in https://github.com/scalar-labs/scalardb/pull/2271/commits/d2d100361ad864defd4d624a1b285fc800424a81